rancher/labels/index.md

```diff
@@ -17,6 +17,8 @@ Key | Value |Description
 `io.rancher.container.hostname_override` | `container_name` | Used to set the hostname of the container to the name of the container (e.g. StackName_ServiceName_CreateIndex)
 `io.rancher.container.start_once` |`true` | Used to run a container once and have it remain in stopped state while the service remains in `active` state
 `io.rancher.container.pull_image` | `always` | Used to always pull a new image before deploying container. 
+`io.rancher.service.selector.container` |  [Selector Format]({{site.baseurl}}/rancher/labels/#selector-format) | Used on a service so that new standalone containers can be selected to join the service DNS. Note: As standalone containers, none of the service actions will affect the standalone container (i.e. deactivate/delete service, restart container, healthcheck, etc). 
```

Comment:
"restart container" should be removed from the list. You can still call individual containers actions to the container. Only service actions - deactivate/update/activate and healtcheck won't take an affect.